armi/physics/neutronics/parameters.py

```diff
@@ -142,7 +143,7 @@ def _getNeutronicsBlockParams():
 
         pb.defParam(
             "mgFluxSK",
-            units="",
+            units=units.UNITLESS,
```

Comment:
I love it!

My preferred approach here is to open a ticket in the downstream repo plugin.  Because (a) that is probably an internal repo, and (b) work will need to be done in this repo AND that one.

Feel free to pass me the name of the repo and I can do that for you.

Reply:
I fixed the units though.